aiohttp/abc.py

```diff
@@ -206,3 +206,18 @@ def log(self,
             response: StreamResponse,
             time: float) -> None:
         """Emit log to logger."""
+
+
+class AbstractAsyncAccessLogger(ABC):
+    """Abstract async writer to access log."""
+
+    def __init__(self, logger: logging.Logger, log_format: str) -> None:
```

Comment:
Drop constructor.
Adding it to `AbstractAccessLogger` was an error :(

Reply:
I can't drop `__init__` since it makes initialising the logger in `RequestHandler.__init__` difficult.

If you have another idea, let me know what you're thinking of.

Reply:
ignore that, I've changed it.